src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * // does not work in Firefox currently due to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037
+ * @param  {} darkQuery
```

Comment:
type missing:
```suggestion
 * @param  {object} darkQuery
```

Reply:
(or maybe `Object` – pls check eslint, don't know it out-of-my head :laughing: )